homeassistant/components/apsystems_local/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""The config_flow for APsystems local API integration."""
+
+from aiohttp import client_exceptions
+from APsystemsEZ1 import APsystemsEZ1M
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_DEVICE_ID, CONF_IP_ADDRESS, CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, LOGGER
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_IP_ADDRESS): str,
+        vol.Required(CONF_NAME): str,
```

Comment:
Why would we ask for the name from a user? Don't we know it's their inverter and can just name it Solar? If that works for 99% of the users, that 1% can rename it later.